### DISCUSSION TITLE
Contributor call: 2 April 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2045] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lucibytes on our Discord server.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 19 March 2022 5pm UTC, which is can be found in your local time zone [here](https://savvytime.com/converter/utc-to-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/Mar-19-2022/5pm)
Or check the bot in Discord! #meetings
**⚠Important note:** timezones have been switching up lately. 
The meetings will always be at 6pm, Antwerp time.

### Agenda
- The newsletter came out!
- Pattern suggestions in Discord (@woutervdub) 

> This discussion will be updated after the call with notes and to-dos.

**Comment by l3xander** [comment, marked as answer]
- The newsletter came out! But there are some issues: some people are complaining and Gmail sends the newsletter to the spambox, apparently because people have reported it as spam. We need to re-address how we’ll handle the next newsletter.
- @woutervdub’s idea of pattern suggestions
    - Summary: we want to find the overlap of what patterns people want and what developers want to make. There are multiple ways we can go about gathering information and displaying it, no concrete decisions have been made on that yet.
    - Someone in Discord mentioned that they couldn’t find the patterns they were looking for. Since we don’t know what people are looking for, we don’t try to cater to that, even though some developers (like @woutervdub) are willing to make these patterns.
    - @joostdecock wants to be cautious about inviting suggestions, there are probably more ‘wanters’ than ‘makers’ and it can’t become a to-do list. We also need to make sure it doesn’t get taken over by trolls. @tangerineshark suggests adding a disclaimer that there’s no guarantee a pattern will get made.
    - For this there are two things we need: to gather information of what people want and to display that information (for people to vote on).
        - It can be done within Discord: @lexander2002 (me) suggested a designated thread or channel (or (Google) Forms) for gathering the information and then putting it in a bot that can make polls. There are also bots specifically made for suggestions.
        @joostdecock mentioned that in this case there should be a link to it on the website, not everyone is in the Discord.
        - @nicholasdower added that putting a link on the site creates more engagement too, instead of people being like “they don’t have the pattern I want, bye,” they will stick around and talk in Discord.
        - There can also be a form (survey) which can then be turned into a list. @joostdecock mentioned @hellgy’s user research, but that seems to be more about the site itself and they are busy at the moment.
        - @bobgeorgethe3rd mentioned that people may give very general suggestions, @joostdecock added that it would help to ask people for a picture of what they mean, this also solves linguistic barriers.
        - @nicholasdower suggests something like the #introduction channel where people don’t talk but only add emojis to other people’s messages. That there will be a list in the server that gets added onto and that people can vote on.
        - @nicholasdower will create a Discussion on this so we can work it out further.
- @joostdecock: it’s on the roadmap to make it easier to extend patterns. When you extend a pattern, you extend the code, but the configuration doesn’t get extended at all and has to be done manually. @joostdecock will make a documentation page on how to do this.
- @joostdecock: in the new .dev site you can choose which version you want a pattern to be drafted in. Now is the question, how far do we go back with versions?
    - Quick rundown: this feature exists so people can compare versions. If there’s a bug, you can find out in which version it broke. 
    @woutervdub asked if it will be possible to compare two different versions within the environment as a new experimental testing feature, but that seems tricky to implement.
    - @starfetch asked what the limiting factor is, what’s the reason we don’t do every version?
    @joostdecock explained that for each version the code needs to be downloaded, so it becomes a lot to have a page for each version for each pattern (build time and bundle time).
    - @bobgeorgethe3rd suggested to add the version where the snap feature was added (one of the final ones of 2.18) and to delete versions that only had small changes to the website and not to the patterns. We decided to go with this solution!
- @bobgeorgethe3rd’s prototype of the Billy bucket hat.
    - The main thing that needs to be done is the reconfiguration of the default settings: the shape is good, but the brim and crown are too big. The pattern does taper upwards, but it’s not enough.
    - It also has to be figured out how to add straps. @darigovresearch suggested on Github to have the holes for the straps be in the code, so people don’t have to decide themselves where to punch the holes. But there are no eyelet hole snippets in existing patterns, only in the documentation, so that’d have to be developed separately.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
